extractor/src/test/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeMixPlaylistExtractorTest.java

```diff
@@ -0,0 +1,241 @@
+package org.schabi.newpipe.extractor.services.youtube.extractors;
+
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.startsWith;
+import static org.junit.Assert.*;
+import static org.schabi.newpipe.extractor.ExtractorAsserts.assertIsSecureUrl;
+import static org.schabi.newpipe.extractor.ServiceList.YouTube;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.schabi.newpipe.DownloaderTestImpl;
+import org.schabi.newpipe.extractor.ListExtractor;
+import org.schabi.newpipe.extractor.ListExtractor.InfoItemsPage;
+import org.schabi.newpipe.extractor.NewPipe;
+import org.schabi.newpipe.extractor.exceptions.ExtractionException;
+import org.schabi.newpipe.extractor.stream.StreamInfoItem;
+
+public class YoutubeMixPlaylistExtractorTest {
+
+    private static YoutubeMixPlaylistExtractor extractor;
+    private static String videoId = "_AzeUSL9lZc";
+    private static String videoTitle = "Most Beautiful And Emotional  Piano: Anime Music Shigatsu wa Kimi no Uso OST IMO";
+
+    public static class Mix {
+
+        @Before
```

Comment:
These should be `@BeforeClass`, otherwise the testing time skyrockets (it just took me 2 minutes just to run this class).

Reply:
Yeah, that makes a lot of sense. That probably is also the reason why the recaptcha are appearing.

Reply:
May I change it or will you do it?

Reply:
@Stypox Just changing it as is, produces NullPointerExceptions in some test. I'll have to look into it.

Reply:
I think you need to add it for all 3 tests.

Reply:
@Stypox I changed it. Was because i had 2 tests for invalid urls, which set the data to null